src/main/java/sirius/biz/tenants/deletion/DeleteTenantJobFactory.java

```diff
@@ -85,7 +85,7 @@ protected String createProcessTitle(Map<String, String> context) {
     @Override
     protected void execute(ProcessContext process) throws Exception {
         Tenant<?> tenant = process.require(TENANT_PARAMETER);
-        boolean takeAction = process.require(DeleteTenantJobFactory.TAKE_ACTION_PARAMETER);
+        boolean takeAction = !process.require(DeleteTenantJobFactory.TAKE_ACTION_PARAMETER);
 
         if (!takeAction) {
```

Comment:
also here, change the parameter to `simulate` and adjust evaluation to avoid double negation

Reply:
please check out if it makes sense now, with the rename.
Otherwise I would have to negate all follwoing ifs..